### PR TITLE
Create documentation for vrrp

### PR DESCRIPTION
**Pull request by rnurgaliyev**

### CODE REVIEW

File: pages/03.Tutorials/17.l3-high-availability/docs.en.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
++--------------------------------------+------+-------------------+-----------------------------------------------------------------------------+--------+
+| db8fa14d-5800-4db5-8ad7-fe9ad1eaef37 |      | fa:16:3e:08:e6:15 | ip_address='10.200.51.34', subnet_id='9a027a94-e688-45cb-a653-e223ccc9f72f' | ACTIVE |
+| 913c4131-0cc3-4df4-9f7e-b4cf4ae885ca |      | fa:16:3e:8d:dc:bb | ip_address='10.200.51.32', subnet_id='9a027a94-e688-45cb-a653-e223ccc9f72f' | ACTIVE |
+| a8389d5e-6bd0-4368-a6fb-249127f9cdca |      | fa:16:3e:c9:e0:b1 | ip_address='10.200.51.1', subnet_id='9a027a94-e688-45cb-a653-e223ccc9f72f'  | ACTIVE |
+| e220e2d9-f4de-4941-92e0-640ee352aa4f |      | fa:16:3e:f3:a9:a3 | ip_address='10.200.51.33', subnet_id='9a027a94-e688-45cb-a653-e223ccc9f72f' | ACTIVE |
++--------------------------------------+------+-------------------+-----------------------------------------------------------------------------+--------+
+```
+
+Second and fourth lines are ports, connecting `ha_first` and `ha_second` accordingly. Now, we must allow 10.200.51.10 on these ports:
```
Comment (dschwabeS11) [inline pages/03.Tutorials/17.l3-high-availability/docs.en.md]: IPs should be shown as code `...`

File: pages/03.Tutorials/17.l3-high-availability/docs.en.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+64 bytes from 10.200.51.10: icmp_seq=3 ttl=64 time=1.03 ms
+
+--- 10.200.51.10 ping statistics ---
+3 packets transmitted, 3 received, 0% packet loss, time 2003ms
+rtt min/avg/max/mdev = 1.035/1.818/3.243/1.009 ms
+
+ubuntu@observer:~$ ip neigh
+10.200.51.10 dev ens3 lladdr 00:00:5e:00:01:01 REACHABLE
+10.200.51.1 dev ens3 lladdr fa:16:3e:c9:e0:b1 REACHABLE
+ubuntu@observer:~$
```
Comment (dschwabeS11) [inline pages/03.Tutorials/17.l3-high-availability/docs.en.md]: Remove line - doesn't show anything useful here. :)

File: pages/03.Tutorials/17.l3-high-availability/docs.en.md [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
+    inet 127.0.0.1/8 scope host lo
+       valid_lft forever preferred_lft forever
+2: ens3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 576 qdisc fq_codel state UP group default qlen 1000
+    inet 10.200.51.32/24 brd 10.200.51.255 scope global dynamic ens3
+       valid_lft 86037sec preferred_lft 86037sec
+3: vrrp.1@ens3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 576 qdisc noqueue state UP group default qlen 1000
+    inet 10.200.51.10/32 scope global vrrp.1
+       valid_lft forever preferred_lft forever
+ubuntu@ha-first:~$
```
Comment (dschwabeS11) [inline pages/03.Tutorials/17.l3-high-availability/docs.en.md]: Remove line - doesn't show anything useful here. :)

File: pages/03.Tutorials/17.l3-high-availability/docs.en.md [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+3 packets transmitted, 3 received, 0% packet loss, time 2003ms
+rtt min/avg/max/mdev = 1.035/1.818/3.243/1.009 ms
+
+ubuntu@observer:~$ ip neigh
+10.200.51.10 dev ens3 lladdr 00:00:5e:00:01:01 REACHABLE
+10.200.51.1 dev ens3 lladdr fa:16:3e:c9:e0:b1 REACHABLE
+ubuntu@observer:~$
+```
+
+We can now sucessfully reach virtual ip address. It has MAC address 00:00:5e:00:01:01, which, according to RFC, is
```
Comment (dschwabeS11) [inline pages/03.Tutorials/17.l3-high-availability/docs.en.md]: *the multiple times

File: pages/03.Tutorials/17.l3-high-availability/docs.en.md [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+virtual mac address assigned to VRRP virtual router 1 (configuration option virtual_router_id)
+
+Let's now bring down keepalived process on `ha_first` and check if virtual IP address will now be served by `ha_second`:
+
+```shell
+ubuntu@ha-first:~$ sudo systemctl stop keepalived.service
+```
+
+We can now see that `ha_second` took the IP address:
```
Comment (dschwabeS11) [inline pages/03.Tutorials/17.l3-high-availability/docs.en.md]: Now we can see...

File: pages/03.Tutorials/17.l3-high-availability/docs.en.md [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+       valid_lft forever preferred_lft forever
+```
+
+There is confirmation in log:
+
+```shell
+Aug 07 09:06:18 ha-second Keepalived_vrrp[8665]: VRRP_Instance(vrrp_1) Transition to MASTER STATE
+Aug 07 09:06:19 ha-second Keepalived_vrrp[8665]: VRRP_Instance(vrrp_1) Entering MASTER STATE
+```
+
```
Comment (dschwabeS11) [inline pages/03.Tutorials/17.l3-high-availability/docs.en.md]: Add `## Conclusion'

File: pages/03.Tutorials/17.l3-high-availability/docs.en.md [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+With this configuration, it takes approximately 2 seconds to discover failure and update MAC forwarding tables.
+Convergence time could be decreased by tuning `Advertisement_Interval`, `Skew_Time` and `Master_Down_Interval`.
```
Comment (dschwabeS11) [inline pages/03.Tutorials/17.l3-high-availability/docs.en.md]: can be

File: pages/03.Tutorials/17.l3-high-availability/docs.en.md (hunk 3 above)
Comment (rnurgaliyev) [reply pages/03.Tutorials/17.l3-high-availability/docs.en.md]: I believe it shows that the output of command is over and considered good practice in documentation.

**Pull request merged**